### DISCUSSION TITLE
这个项目可以获取文件的哈希值吗？

### DISCUSSION THREAD

**Discussion by tobexeon** [messense/aliyundrive-webdav#331] [Q&A] [answered]
使用rclone check时，提示无法获取文件哈希值，但是我记得阿里云盘是提供文件sha1的呀

**Comment by messense** [comment, marked as answer]
WebDAV 标准协议中没有 checksum

https://datatracker.ietf.org/doc/html/rfc4918

**Reply by messense** [reply]
增加了 [owncloud/nextcloud 风格的 checksums 支持](https://github.com/messense/aliyundrive-webdav/pull/524)，rclone webdav 可以试试 owncloud 模式。
